llvm/test/CodeGen/X86/machine-block-freq.ll

```diff
@@ -0,0 +1,44 @@
+; RUN: llc -mtriple=x86_64-linux-gnu -stop-after=x86-isel %s -o - | llc --passes='print<machine-block-freq>' -x mir -o - 2>&1 | FileCheck %s
```

Comment:
Use -filetype=null if you are just discarding the output.

Is there much point to starting with IR if you're just codegening it to get the MIR?

Reply:
Convert to MIR test for now. Because ideally these test should stay in `test/CodeGen/Generic`, but I couldn't find a good way to write a generic test, `llc` may crash with its output on some targets.

Reply:
There is no such thing as a generic codegen test. Don't put anything in there, we should move just about everything in there out that's there now